src/Http/Controllers/LogViewerController.php

```diff
@@ -148,14 +148,10 @@ public function search(Request $request, $date, $level = 'all')
 
         $log     = $this->getLogOrFail($date);
         $levels  = $this->logViewer->levelsNames();
-        $needles = array_filter(explode(' ', $query));
-        $entries = $log->entries($level)
-            ->unless(empty($needles), function (LogEntryCollection $entries) use ($needles) {
-                return $entries->filter(function (LogEntry $entry) use ($needles) {
-                    return Str::containsAll($entry->header, $needles);
-                });
-            })
-            ->paginate($this->perPage);
+        $needles = array_map('strtolower', explode(' ', $query));
+        $entries = $log->entries($level)->filter(function ( LogEntry $value) use ($needles) {
```

Comment:
Can you remove the space from `function ( LogEntry $value)` => `function (LogEntry $value)`

Reply:
yes done